docs/ref/models/lookups.txt

```diff
@@ -177,16 +177,22 @@ following methods:
     comparison between ``lhs`` and ``rhs`` such as ``lhs in rhs`` or
     ``lhs > rhs``.
 
-    The notation to use a lookup in an expression is
+    The primary notation to use a lookup in an expression is
     ``<lhs>__<lookup_name>=<rhs>``.
 
-    This class acts as a query expression, but, since it has ``=<rhs>`` on its
-    construction, lookups must always be the end of a lookup expression.
+    .. versionchanged:: 4.0
+
+        A lookup can also be used directly as an expression:
```

Comment:
```suggestion
        A lookup can also be used directly as an expression::
```

Think this might fix the issue with the docs not building.

Reply:
Doesn't look like it :disappointed: